src/docs/06_runtime_view.adoc

```diff
@@ -1,54 +1,117 @@
 [[section-runtime-view]]
+
 == Runtime View
 
+=== Login
 
-[role="arc42help"]
-****
-.Contents
-The runtime view describes concrete behavior and interactions of the system’s building blocks in form of scenarios from the following areas:
+1. The user selects it's pod provider
+2. The user types it's personal id and password
```

Comment:
Here it should be "its" instead of "it's".

Reply:
Fixed